docs/deployment/deployment-tasks.md

```diff
@@ -0,0 +1,54 @@
+# Deployment Tasks
+
+Sometimes you need to run a command on at deployment time, but before an app is completely deployed.
+Common use cases include:
+
+* Checking a database is initialised
+* Running database migrations
+* Any commands required to set up the server (e.g. something like a Django `collectstatic`)
+
+## `app.json` and `scripts.dokku`
+
+Dokku accomplishes this by using an `app.json` file. We (mostly) use the same format as Heroku's [app.json](https://devcenter.heroku.com/articles/app-json-schema).
+However, dokku currently only supports the nodes `scripts.dokku.predeploy` and `scripts.dokku.postdeploy`.
```

Comment:
Maybe. It feels a little confusing from a user perspective though. As that wasn't part of the original issue (#1824), I propose we punt on this one and see if we get any user feedback asking for it.